### PR TITLE
feat(users): list users via the API now supports `offset`, `limit` or `after` parameter

### LINKED ISSUE #21402: Add pagination support to `/users` API
Add support for `limit`/`after`/`descending` to the `GET /users` API.

### PR DESCRIPTION
**Pull request by bednar**
Closes #21402
Related to https://github.com/influxdata/influxdb-client-csharp/issues/190

List users via the API now supports `offset`, `limit` or `after` parameter

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [x] http/swagger.yml updated (if modified Go structs or API)
- [x] Documentation updated or issue created (provide link to issue/pr)
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: http/swagger.yml [hunk 1]
```diff
@@ -5395,6 +5395,9 @@ paths:
       summary: List all users
       parameters:
         - $ref: "#/components/parameters/TraceSpan"
+        - $ref: "#/components/parameters/Offset"
+        - $ref: "#/components/parameters/Limit"
+        - $ref: "#/components/parameters/After"
```
Comment (glinton) [inline http/swagger.yml:5400]: These swagger updates should be made to the openapi repo

Comment (bednar) [reply http/swagger.yml]: Changes are moved into `openapi` repository: https://github.com/influxdata/openapi/pull/44

**Comment by danxmoran** [comment]
Hrm, I think the build failure is an unrelated bug in our logic to detect & avoid running certain CI steps from fork-PRs... will investigate

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 1. [19811](https://github.com/influxdata/influxdb/pull/19811): Add Geo graph type to be able to store in Dashboard cells.
 1. [21218](https://github.com/influxdata/influxdb/pull/21218): Add the properties of a static legend for line graphs and band plots.
+1. [21367](https://github.com/influxdata/influxdb/pull/21367): List users via the API now supports `offset`, `limit` or `after` parameter.
```
[the remark this answers is not in the document]
Comment (bednar) [reply CHANGELOG.md]: I've changed it to general version: `List users via the API now supports pagination`.

**Pull request merged**